src/app/results/results.component.ts

```diff
@@ -188,5 +188,6 @@ export class ResultsComponent implements OnInit {
 
   ngOnInit() {
     document.getElementById('nav-group').style.width = '632px';
+    document.getElementById('nav-input').style.width = '536px';
```

Comment:
this can be an acceptable solution but try using as much as typescript than javascript.and let code look like angular code.
for instance you can use attribute binding `[style.width]="navbarwidth"` for the element and set `this.navbar=536px` 
https://angular.io/guide/structural-directives